test/integration/test/webpush_test.js

```diff
@@ -0,0 +1,165 @@
+'use strict';
+
+const chakram = require('chakram'), expect = chakram.expect;
+const Config = require('config-js');
+
+var config = new Config('./test/integration/lib/config/foxbox.js');
+var Prepper = require('../lib/testPrepper.js');
+
+Prepper.makeSuite('Test Push Service locally',function(){
+  var header = new Config('./test/integration/lib/config/header.js');
+  var credential = config.get('credential'); 
+
+  var setupURL = config.get('foxbox.url') + '/users/setup';
+  var loginURL = config.get('foxbox.url') + '/users/login';
+  var serviceURL = config.get('foxbox.url') + '/api/v1';
+  var serviceListURL = serviceURL + '/services'; 
+  var getterURL = serviceURL + '/channels/get';
+  var setterURL = serviceURL + '/channels/set';
+    
+  it('Create a new credential',function(){
```

Comment:
I don't recall exactly if I asked a similar question already. Why do we have to check the new credentials are created? Couldn't this be done in a `before()`?